### PR TITLE
[TACHYON-784] [TACHYON-743] Variable re-order and import cleanup

### PR DESCRIPTION
**Pull request by rean**
Small changes, please review.

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 1]
```diff
@@ -50,7 +50,8 @@
  */
 public class BlockMasterSync implements Runnable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
-
+  /** The thread pool to remove block */
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: Could you leave this comment back with the 'mFixedExecutionService' variable?

**Comment by gpang** [comment]
Thanks for the cleanup! I just left a minor comment.

**Comment by rean** [comment]
thanks for taking a look. I moved the comment back with the mFixedExecutionService variable.

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM

**Comment by haoyuan** [comment]
Thanks @rean ! Do you have an account on JIRA (https://tachyon.atlassian.net/)? If so, we can assign these two tasks to you and then close them.

**Comment by rean** [comment]
I just created an account on jira, the username is rean.

**Comment by haoyuan** [comment]
Thanks!

**Pull request merged**